style.css

```diff
@@ -76,22 +76,24 @@ body{
 }
 
 #preview{
-    flex: 15%;
-    display: flex;
-    flex-direction: column;
+    /*flex-direction: column;
+    font-size: 2.5vw; (erro de digitação(?)/argumento sobrescrito por estilos mais abaixo na cascata)
+    flex: 15%;*/
+    display: block;
     align-items: end;
     position: relative;
-    padding: .1em .2em;
+    padding: 0.45vw .5vw;
     background-color: white;
-    font-size: 2.5vw;
```

Comment:
Font-size property was set in terms of viewport width; this was set as a comment, **should** be removed later.